hpat/tests/test_series.py

```diff
@@ -3010,6 +3018,9 @@ def test_series_sort_values_full_unicode4(self):
         def test_impl(series, ascending):
             return series.sort_values(axis=0, ascending=ascending, inplace=False, kind='quicksort', na_position='last')
 
+        def pandas_impl(series, ascending):
+            return series.sort_values(axis=0, ascending=ascending, inplace=False, kind='mergesort', na_position='last')
```

Comment:
Why not pass kind of sorting to the implementation as argument to have a single implementation?

Reply:
Fixed